### ISSUE TITLE
Is it possible to add a re-read/replay function 

### ISSUE THREAD

**Issue by themergoger** [C-Nedelcu/talk-to-chatgpt#145]
Hello, @C-Nedelcu! I want to express my gratitude for this wonderful bot. It has truly brought Chat-GPT to life, and I greatly appreciate it. I do have a request, if possible. Could you consider adding a reread function? This would allow the bot to repeat the last block of text without me having to ask for a new response. Sometimes, I might miss what was said or get distracted, and having the ability to hear the answer again would be incredibly helpful. It would also be convenient for sharing the response with others. Thank you so much for your hard work. I truly appreciate it.
Again Thank you

**Comment by cburfict** [comment]
Agreed! Great plugin but could definitely use a reread/replay button
